gp2gp_adaptor_response_docs.yaml

```diff
@@ -0,0 +1,557 @@
+
+                            examples:
+                                successful-migration:
+                                    summary: Successful Migration scenario
+                                    value:
+                                        - initialRequestTimestamp: "2023-09-20T11:47:58.966Z"
+                                          actionCompletedTimestamp: "2023-09-20T11:54:19.552Z"
+                                          nhsNumber: "9729734925"
+                                          conversationId: "59B118DB-70C3-4883-8A60-5E725981F003"
+                                          migrationStatus: "COMPLETE"
+                                          fromAsid: "858000001001"
+                                          toAsid: "200000001908"
+                                          fromOdsCode: "C88046"
+                                          toOdsCode: "B84012"
+                                failed-migration:
+                                    summary: Failed Migration scenario
+                                    value:
+                                        - initialRequestTimestamp: "2023-09-20T12:18:10.364Z"
+                                          actionCompletedTimestamp: "2023-09-20T12:18:13.923Z"
+                                          nhsNumber: "9729735336"
+                                          conversationId: "C2A59970-57AF-11EE-AFE6-CD607DC58E3B"
+                                          migrationStatus: "FAILED_INCUMBENT"
+                                          fromAsid: "200000000169"
+                                          toAsid: "200000001908"
+                                          fromOdsCode: "P84009"
+                                          toOdsCode: "B84012"
+
+    /ehr-resend/{conversationId}:
+        post:
+            tags:
+                - Ehr resend controller
+            summary: Schedule Ehr Extract Resend
+            description: Schedule Ehr Extract Resend
+            operationId: scheduleEhrExtractResend
+            parameters:
+                - name: conversationId
+                  in: path
+                  required: true
+                  schema:
+                      type: string
+            responses:
+                '202':
+                    description: Successfull Ehr Extract resend request
+                    content:
+                        application/json:
+                            schema:
+                                type: string
+                '404':
+                    description: Not Found
+                    content:
+                        application/json:
+                            schema:
+                                type: string
+                '409':
+                    description: Ehr Extract resend action cannot be triggered
+                    content:
+                        application/json:
+                            schema:
+                                type: string
```

Comment:
For both this and the one above, it would be nice if there was a way to specify that an `OperationOutcome` is returned.